modules/placement-driver/src/main/java/org/apache/ignite/internal/placementdriver/LeaseUpdater.java

```diff
@@ -395,4 +407,54 @@ private boolean isLeaseOutdated(Lease lease) {
             return now.after(lease.getExpirationTime());
         }
     }
+
+    /**
+     * Message handler to process notification from replica side.
+     */
+    private class PlacementDriverActorMessageHandler implements NetworkMessageHandler {
+        @Override
+        public void onReceived(NetworkMessage msg0, String sender, @Nullable Long correlationId) {
+            if (!(msg0 instanceof PlacementDriverActorMessage)) {
+                return;
+            }
+
+            var msg = (PlacementDriverActorMessage) msg0;
+
+            if (!stateActorLock.enterBusy()) {
+                return;
+            }
+
```

Comment:
So we just throw this message away if this is not an active actor? Imagine following:
- active actor crashes
- PD group leader election is in progress
- some group grp0 loses majority and wants to notify PD about this
- every known PD node throws away the message
- PD elects leader fast enough, so the lease for grp0 is not outdated and the same leaseholder is chosen.

So the lease would be prolonged as if nothing happened, right?